velox/common/caching/SsdFile.cpp

```diff
@@ -883,18 +870,14 @@ void SsdFile::initializeCheckpoint() {
         getCheckpointFilePath());
   }
   const auto logPath = getEvictLogFilePath();
-  evictLogFd_ = ::open(logPath.c_str(), O_CREAT | O_RDWR, S_IRUSR | S_IWUSR);
-  if (disableFileCow_) {
-    disableCow(evictLogFd_);
-  }
-  if (evictLogFd_ < 0) {
+  filesystems::FileOptions evictLogFileOptions;
+  evictLogFileOptions.shouldThrowOnFileAlreadyExists = false;
+  try {
```

Comment:
Consider to provide a macro in followup
```
#define FILE_CALL(fileFunction, errorCallback);
```
errorCallback expects a std::exception and both fileFunction and errorCallback could be a lambda function to capture requires execution context.